### PR TITLE
Add a guide for enabling EU endpoint for v2

### PR DESCRIPTION
**Pull request by greenberga**
I adapted the guide for v1. I also changed v1 a bit to use some Docusaurus components.

### CODE REVIEW

File: docs/guides/eu-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Fallback to global service
+
+Although we work hard to make sure it never happens, disaster may one day strike (e.g. a meteor strike to our German data center). In case our EU endpoint service goes down you can instruct your widget to use the global service as a fallback.
+
+You can do this by specifying both endpoints separated with a comma (`,`) in order of preference:
+
+```html
+<div class="frc-captcha" data-sitekey="<my sitekey>" data-api-endpoint="eu,global"></div>
+```
```
Comment (greenberga) [inline docs/guides/eu-endpoint.md:60]: Is this true?

File: docs/guides/eu-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## EU Verification Endpoint
+
+Your servers can also use our EU endpoint for the verification of submitted challenge solutions.
+
+Instead of the [usual verification endpoint](../api/siteverify) your server makes the POST request to `https://eu.frcapi.com/api/v2/captcha/siteverify`.
```
Comment (greenberga) [inline docs/guides/eu-endpoint.md:66]: Is this true?

File: docs/guides/eu-endpoint.md (hunk 1 above)
Comment (gzuidhof) [reply docs/guides/eu-endpoint.md:60]: v2 doesn't have this fallback mechanism, it shouldn't be necessary (it is much more redundant and multi-region). So I think we can remove this section entirely.

File: docs/guides/eu-endpoint.md (hunk 2 above)
Comment (gzuidhof) [reply docs/guides/eu-endpoint.md:66]: Yes :).

At this point one can use either `global.frcapi.com` or `eu.frcapi.com` (it doesn't have to match the one used in the widget). There is not much of a functional difference (and generally the server IP address is not PII which removes the main reason for favoring the EU endpoint).

File: docs/guides/eu-endpoint.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+https://global.frcapi.com/api/v2/captcha/siteverify
+```
+
+### 🇪🇺 EU
+
+```
+https://eu.frcapi.com/api/v2/captcha/siteverify
+```
+
+When configuring the widget in your front-end, can you can just specify `eu` or `global`.
```
Comment (gzuidhof) [inline docs/guides/eu-endpoint.md:84]:
```suggestion
When configuring the widget in your front-end, you can specify the shorthand `eu` or `global`.
```

File: docs/guides/eu-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+## Troubleshooting
+
+If your widget or the browser console shows **Endpoint not allowed** or **403 Forbidden**, double-check that the you enabled the configured endpoint for the given sitekey.
+
+If you run into any other issues you can of course always reach out.
```
Comment (gzuidhof) [inline docs/guides/eu-endpoint.md:90]:
```suggestion
```

I think we don't have to explicitly mention it, I hope customers are aware they can always reach out

**Review by gzuidhof — APPROVED** [review]

**Pull request merged**